meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend

```diff
@@ -16,6 +16,7 @@ PACKAGECONFIG[ubootenv] = ",,u-boot-fw-utils,u-boot-fw-utils u-boot-default-env
 PACKAGECONFIG[libfyaml] = ",,,libfyaml"
 PACKAGECONFIG[aklite-offline] = "-DBUILD_AKLITE_OFFLINE=ON,-DBUILD_AKLITE_OFFLINE=OFF,"
 PACKAGECONFIG[nerdctl] = "-DBUILD_AKLITE_WITH_NERDCTL=ON,-DBUILD_AKLITE_WITH_NERDCTL=OFF,,nerdctl"
+PACKAGECONFIG[hsm] = "-DBUILD_P11=ON -DPKCS11_ENGINE_PATH=${PKCS11_ENGINE_PATH},-DBUILD_P11=OFF,libp11,aktualizr-pkcs11-label"
```

Comment:
This there a `+=` or `_append` type thing we can do so it doesn't drift from the source - https://github.com/uptane/meta-updater/blob/446ea73443a7ad69dc2db43428aeff67454c8188/recipes-sota/aktualizr/aktualizr_git.bb#L61

also - should we try and upstream this into meta-updater instead?

Reply:
+= doesn't work in this case. I'm also worried about missing changes in meta-updater. But I guess upstreaming is the long term plan. It should happen for all patches related to pkcs11 token label so it might take some time.

Reply:
I think they'd be willing to take this patch right now.

Reply:
Once the label logic is merged upstream we can land in meta-updater.

For now the way is indeed to replace the entire line, as we can't append here.